File: Java/Tests/jdi-uitest-webtests/src/test/java/com/epam/jdi/uitests/testing/unittests/InitTests.java
package com.epam.jdi.uitests.testing.unittests;

import com.epam.jdi.uitests.testing.unittests.entities.User;
import com.epam.jdi.uitests.testing.unittests.pageobjects.EpamJDISite;
import com.epam.jdi.uitests.web.selenium.elements.composite.WebSite;
import com.epam.jdi.uitests.web.testng.testRunner.TestNGBase;
import com.epam.web.matcher.verify.Verify;
import org.testng.annotations.AfterMethod;
import org.testng.annotations.BeforeSuite;

import static com.epam.jdi.uitests.testing.unittests.pageobjects.EpamJDISite.homePage;
import static com.epam.jdi.uitests.testing.unittests.pageobjects.EpamJDISite.login;
import static com.epam.jdi.uitests.web.settings.WebSettings.*;


/**
 * Created by Roman_Iovlev on 7/13/2015.
 */
public class InitTests extends TestNGBase {

    @BeforeSuite(alwaysRun = true)
    public static void setUp() throws Exception {
        WebSite.init(EpamJDISite.class);
        homePage.open();
        login.submit(User.DEFAULT);
        Verify.getFails();
        logger.info("Run Tests");
    }

    @AfterMethod
    public void tearDown() {
        Verify.getFails();
    }
}

File: Python/tests/init_tests.py
import unittest

from JDI.web.selenium.settings.WebSettings import WebSettings
from tests.site.pageobjects.epam_jdi_site import EpamJDISite
from JDI.web.selenium.elements.composite.web_site import WebSite
from tests.site.entities.user import User


class InitTests(unittest.TestCase):

    @classmethod
    def setUpClass(cls):
        WebSite.init(EpamJDISite, WebSettings.use_driver("chrome"))
        WebSettings.logger.info("Run Tests")
        EpamJDISite.home_page.open()
        EpamJDISite.login_page.submit(User.default())
        # WebSettings.get_logger().info("Run Tests")

    @classmethod
    def tearDownClass(cls):
        WebSettings.quit_browser()
